c2corg_ui/templates/helpers/view.html

```diff
@@ -111,7 +111,8 @@
       title = ''
       if 'title_prefix' in locale and locale['title_prefix']:
           title = locale['title_prefix'] + ' : '
-      title += locale['title']
+      if 'title' in locale and locale['title']:
+          title += locale['title']
```

Comment:
I think that change was made by an upset user that wanted to sabotage the route...